

// purposeful error
def FOO: Void = 12;